client/src/App.css

```diff
@@ -5,7 +5,7 @@
 .roundButton {
   background-color: #182C4A;
   color: #fff;
-  border:1px solid #182C4A;
```

Comment:
Were these spacings added for readability?

Reply:
Oh I think my vs code settings does that automatically